server/join_test.go

```diff
@@ -133,14 +133,13 @@ func mustNewJoinCluster(c *C, num int) ([]*Config, []*Server, cleanUpFunc) {
 	svrs := make([]*Server, 0, num)
 	cfgs := newTestMultiJoinConfig(num)
 
-	for _, cfg := range cfgs {
+	for i, cfg := range cfgs {
 		svr, err := startPdWith(cfg)
 		c.Assert(err, IsNil)
 		svrs = append(svrs, svr)
+		waitMembers(svrs[0], i+1)
```

Comment:
We have to add member one by one, otherwise cluster may reports:"unhealthy cluster" beacuse the last added member is not ready.